torch/distributed/_tensor/dispatch.py

```diff
@@ -86,6 +87,8 @@ def __init__(self) -> None:
             aten.is_same_size.default: is_same_size_handler,
             aten.convolution.default: convolution_handler,
             aten.convolution_backward.default: convolution_backward_handler,
+            # aten.scaled_dot_product_attention.default: sdpa_handler,
+            aten._scaled_dot_product_flash_attention.default: sdpa_handler,
```

Comment:
IMO we should not by default enable this, until we figure out a way to decide when to apply ring attention and when not. Given this is pretty experimental, what we should probably do:
1. put attention.py to `_tensor/experimental` folder, we can move to non-experimental later if we figured out the UX.
2. probably add a context manager similar to [loss_parallel](https://github.com/pytorch/pytorch/blob/main/torch/distributed/tensor/parallel/loss.py#L27) that only enables the sdpa_handler when the ctx manager activated

Reply:
done